### ISSUE TITLE
Game crashes when trying to load a map

### ISSUE THREAD

**Issue by avril-beloved** [Luffyyy/BeardLib-Editor#768]
### Describe the crash

Yep, the game crashes when I try to load any map to edit it. I checked the crashlog and this appeared: 

Sat Mar 11 22:19:42 2023

Application has crashed: C++ exception
mods/BeardLib-Editor-master/Classes/MissionElementUnit.lua:81: attempt to index field 'EditorOptions' (a nil value)

SCRIPT STACK

create_mission_element_unit() @mods/BeardLib-Editor-master/Hooks/MissionManager.lua:574
create_element() @mods/BeardLib-Editor-master/Hooks/MissionManager.lua:540
_create_elements() @mods/BeardLib-Editor-master/Hooks/MissionManager.lua:521
original() core/lib/managers/mission/coremissionmanager.lua:403
init() @mods/base/req/core/Hooks.lua:262
new() core/lib/utils/coreclass.lua:35
_add_script() core/lib/managers/mission/coremissionmanager.lua:113
_load_mission_file() @mods/BeardLib-Editor-master/Hooks/MissionManager.lua:130
parse() @mods/BeardLib-Editor-master/Hooks/MissionManager.lua:40
init_game() lib/setups/gamesetup.lua:740
core/lib/setups/coresetup.lua:424

I don't know what happened or how to fix it. I have everything downloaded correctly so I don't really know what's going on, please help

### Steps to reproduce the crash

Just load into any, and I mean literally ANY map (custom or not) and it will crash the game

### Logs

Sat Mar 11 22:19:42 2023

Application has crashed: C++ exception
mods/BeardLib-Editor-master/Classes/MissionElementUnit.lua:81: attempt to index field 'EditorOptions' (a nil value)

SCRIPT STACK

create_mission_element_unit() @mods/BeardLib-Editor-master/Hooks/MissionManager.lua:574
create_element() @mods/BeardLib-Editor-master/Hooks/MissionManager.lua:540
_create_elements() @mods/BeardLib-Editor-master/Hooks/MissionManager.lua:521
original() core/lib/managers/mission/coremissionmanager.lua:403
init() @mods/base/req/core/Hooks.lua:262
new() core/lib/utils/coreclass.lua:35
_add_script() core/lib/managers/mission/coremissionmanager.lua:113
_load_mission_file() @mods/BeardLib-Editor-master/Hooks/MissionManager.lua:130
parse() @mods/BeardLib-Editor-master/Hooks/MissionManager.lua:40
init_game() lib/setups/gamesetup.lua:740
core/lib/setups/coresetup.lua:424

-------------------------------

Callstack:

         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     zip_get_name                                        
                         ntdll  (???)     RtlAllocateHeap                                     

-------------------------------

Current thread: Main

-------------------------------

System information:
	Application version : 1.136.176
	CPU : Intel(R) Core(TM) i3-10110U CPU @ 2.10GHz (1 core); SSE; SSE2; SSE3; SSSE3; SSE4.1; SSE4.2
	DirectX : 12.0 
	GPU : Intel(R) UHD Graphics / igdumdim32.dll[27.20.100.8729]
	Language : english
	Memory :     12111MB 628KB
	OS : 6.2.9200 () 0x300-0x1 (64 bits)
	Physics : threaded
	Renderer : DX9 threaded
	Sound : Realtek Semiconductor Corp. (Speaker (Realtek(R) Audio))

**Comment by Xeletron** [comment]
Can you send your logs after you get this crash? you can find it in `mods/logs/[todays date]`
Seems like something might not been able to initialize correctly

**Comment by avril-beloved** [comment]
> 
09:32:57 AM FATAL ERROR:  (C:\projects\payday2-superblt\src\InitiateState.cpp:248) mods/Full Speed Swarm/lua/coresoundenvironmentmanager.lua:16: bad argument #1 to 'max' (number expected, got nil)
stack traceback:
	[C]: in function 'max'
	mods/Full Speed Swarm/lua/coresoundenvironmentmanager.lua:16: in function 'add_check_object'
	mods/BeardLib-Editor-master/Core.lua:131: in function 'func'
	mods/Full Speed Swarm/lua/blt_hooks.lua:32: in function 'Call'
	mods/base/lua/MenuManager.lua:79: in function '_base_process_menu'
	mods/base/lua/MenuManager.lua:55: in function <mods/base/lua/MenuManager.lua:32>
	[C]: in function 'pcall'
	mods/base/lua/MenuManager.lua:32: in function 'func'
	mods/Full Speed Swarm/lua/blt_hooks.lua:32: in function 'Call'
	mods/base/lua/MenuManager.lua:5: in function 'func'
	mods/base/req/core/Hooks.lua:269: in function 'init'
	[string "core/lib/utils/coreclass.lua"]:35: in function 'new'
	[string "lib/setups/setup.lua"]:404: in function 'original'
	mods/base/req/core/Hooks.lua:262: in function 'init_managers'
	[string "lib/setups/gamesetup.lua"]:660: in function 'init_managers'
	[string "lib/setups/networkgamesetup.lua"]:10: in function 'init_managers'
	[string "core/lib/setups/coresetup.lua"]:401: in function <[string "core/lib/setups/coresetup.lua"]:295>

09:32:57 AM Lua: [ERROR] nil 
09:33:03 AM Log:  (C:\projects\payday2-superblt\src\http\http.cpp:44) CURL CLOSED


This thing?

**Comment by Xeletron** [comment]
Yes, looks like full speed swarm might be causing compatibility issues, try uninstalling it and see if it works
